R/validateSettings.R

```diff
@@ -39,24 +39,40 @@
 
 
 validateSettings <- function(data, settings, chart="eDish"){
-
+  
   settingStatus<-list()
-
+  
   # Check that all required parameters are not null
   requiredChecks <- getRequiredSettings(chart = chart) %>% purrr::map(checkRequired, settings = settings)
-
+  
   #Check that non-null setting columns are found in the data
   dataKeys <- getSettingsMetadata(charts=chart, filter_expr = .data$column_mapping, cols = "text_key")%>%textKeysToList()
+  
+  # Add filters and group_cols to dataKeys manually as temporary fix - they are no longer True for column_mapping in settingsMetadata
```

Comment:
I like this fix, but lets automate it to cover all metadata settings where `type=vector` and `column_mapping=true`

Reply:
Which means we need to change `column_mapping` back to true in settingsMetadata.

Reply:
This can probably serve as a resolution for #170 if we apply similar logic to fields